indico/modules/events/registration/models/registrations.py

```diff
@@ -452,6 +452,16 @@ def avatar_url(self):
         """Return the url of the user's avatar."""
         return url_for('event_registration.registration_avatar', self)
 
+    @property
+    def picture_url(self):
+        """Return the url of the picture in personal data."""
+        picture = [d for d in self.data if d.field_data.field.personal_data_type is not None
+                   and d.field_data.field.input_type == 'picture']
+        if picture and picture[0].filename:
+            return url_for('event_registration.registration_picture', self,
```

Comment:
I don't think you can reliably define a property for this. It heavily depends on the context which URL to use (for example if you're viewing your own registration you use the `self.locator.registrant`, but as a manger you wouldn't do that.

I recommend you don't add such a property now, and when you add something for the participant list you create a new endpoint (needed anyway since it'll have to check if the current user can see that person in the participant list and the picture is supposed to be shown there) and use that then (whether this property then makes sense or not is to be seen).

Reply:
Property removed.